Example/Podfile.lock

```diff
@@ -1,23 +1,23 @@
 PODS:
-  - RxCocoa (3.6.1):
-    - RxSwift (~> 3.6)
-  - RxGesture (1.1.0):
-    - RxCocoa (~> 3.6)
-    - RxSwift (~> 3.6)
-  - RxSwift (3.6.1)
+  - RxCocoa (4.0.0-alpha.1):
+    - RxSwift (~> 4.0.0-alpha.1)
+  - RxGesture (1.1.1):
+    - RxCocoa (~> 4.0.0-alpha.0)
+    - RxSwift (~> 4.0.0-alpha.0)
+  - RxSwift (4.0.0-alpha.1)
```

Comment:
There's a `4.0.0-beta.0` of RxSwift now. Can you update for it please?

Reply:
@sidmani Can you `pod update` please in order to update the `Podfile.lock` file?